CHANGES.md

```diff
@@ -47,6 +47,13 @@ OpenSSL 3.4
 
    *Tim Perry*
 
+ * Added to TLS v1.3 support for integrity-only cipher suites
+   TLS_SHA256_SHA256 and TLS_SHA384_SHA384, as defined in RFC 9150.
```

Comment:
Nit: Please reword to something like `Added support for integrity-only cipher suites
   TLS_SHA256_SHA256 and TLS_SHA384_SHA384 in TLS 1.3, as defined in RFC 9150.`